src/style.css

```diff
@@ -0,0 +1,131 @@
+/* #region card-page */
+.card-page {
+  font-family: "Roboto", sans-serif;
+}
+
+.card-page__content {
+  margin: 0;
+}
+
+/* #endregion */
+
+/* #region card */
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  padding: 32px 19px 0;
+}
+
+.card__name {
+  /* display: flex; */
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin-top: 40px;
+  margin-bottom: 0;
+  padding: 0 16px;
+}
+
+.card__code {
+  font-size: 10px;
+  color: #616070;
+  margin: 4px 0 0;
+  padding: 0 16px;
+}
+
+/* #endregion */
+
+/* #region stars */
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 16px 0 0;
+  padding: 0 16px;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.stars_star {
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  background-position: 50%;
+  margin-right: 4px;
+}
+
+.stars--4 .stars_star:nth-child(-n+4) {
+  background-image: url("images/star-active.svg");
+}
```

Comment:
pay attention to this requirement
![image](https://user-images.githubusercontent.com/98764641/226844549-ae96342b-86ef-41ee-b032-84422cd3129e.png)